### DISCUSSION TITLE
Program 'winget.exe' failed to run: The file cannot be accessed by the system.

### DISCUSSION THREAD

**Discussion by ramsoft-inc** [microsoft/winget-cli#1202] [Q&A] [answered]
While running winget command from **windows service** I got the error  `Program 'winget.exe' failed to run: The file cannot be accessed by the system.`

I have read the https://github.com/microsoft/winget-cli/discussions/962#discussioncomment-766620 , I am not whether it is related or not with the same issue.

Can you please update us the status for this issue ?

**Comment by denelon** [comment, marked as answer]
The App Installer MSIX package doesn't get registered for the service account. We're still discussing how this should be addressed, and what possible steps might address the issue here.

**Reply by ramsoft-inc** [reply]
Thank you @denelon , I hope we see this feature very soon.
